### ISSUE TITLE
Parabolic minimum/maximum in impute.quadratic

### ISSUE THREAD

**Issue by SaremS** [stefvanbuuren/mice#66]
Hi,

we are currently doing a group presentation on Polynomial Combination and are quite certain that in line 100 of mice.impute.quadratic the term should be 	y.min <- -b1 / (2 * b2) instead of 	y.min <- -b1 / 2 * b2.

Kind regards,
Sarem

**Comment by gerkovink** [comment]
Hi Sarem, 

Thanks for noticing. This issue has been fixed in pull request #64. In the meantime, if you'd like to run the updated `mice.impute.quadratic()`, you can instruct `R` with `devtools::install_github("gerkovink/mice")` to install an updated version of mice that is ahead of `stefvanbuuren/mice` on this issue. Don't forget to revert back to `stefvanbuuren/mice` via `devtools::install_github("stefvanbuuren/mice")`. 

All the best, 

Gerko

**Comment by SaremS** [comment]
Hi Gerko,

apologies, that strange behaviour was due to a programming error that I
made.
One last issue that we are struggling with: We did an R simulation to show
the superiority of Polynomial Combination against Transform, then Impute
using the same imputation method that you are describing in your 2013 paper
about P.C. Only difference is that we want to vary the beta_1/beta_2 slopes
of the data generating process with every iteration to get a little closer
to real world examples where population slopes likely won't be 1 and 1.
What we find is that the further the beta_2 slope is away from 0, the
larger the gap between full data slopes and imputed slopes will become. We
have tried to scale the variable using z-transformation but that does not
seem to fix the issue for us.

If you find the time for some quick advice here, that would be really
helpful to us.

Thanks a lot,
Sarem

2018-01-24 23:00 GMT+01:00 Gerko Vink <notifications@github.com>:

**Comment by SaremS** [comment]
Hi Gerko,

apologies, that strange behaviour was due to a programming error that I
made.
One last issue that we are struggling with: We did an R simulation to show
the superiority of Polynomial Combination against Transform, then Impute
using the same imputation method that you are describing in your 2013 paper
about P.C. Only difference is that we want to vary the beta_1/beta_2 slopes
of the data generating process with every iteration to get a little closer
to real world examples where population slopes likely won't be 1 and 1.
What we find is that the further the beta_2 slope is away from 0, the
larger the gap between full data slopes and imputed slopes will become. We
have tried to scale the variable using z-transformation but that does not
seem to fix the issue for us.

If you find the time for some quick advice here, that would be really
helpful to us.

Thanks a lot,
Sarem


2018-01-24 23:00 GMT+01:00 Gerko Vink <notifications@github.com>:
